### PR TITLE
Basic react big component for homeYouMayLike module

### PR DESCRIPTION
**Pull request by galw**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,121 @@
...
+        </div>
+
+        <div class="reviews">
+          <span class="star">&#9733</span>
+          <span class="rating">4.54</span>
+          <span class="totalReviews">(171)</span>
+        </div>
+      </div>
+    </div>
+  </div>
```
Comment (cshih1141) [inline public/index.html]: I'm a little confused as to why you have all this code here as well as your index.jsx.

File: client/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+            <div className="price">
+              <span>$200/night</span>
+            </div>
+
+            <div className="reviews">
+              <span className="star">&#9733</span>
+              <span className="rating">4.54</span>
+              <span className="totalReviews">(171)</span>
+            </div>
```
Comment (cshih1141) [inline client/src/index.jsx]: you should think about modularizing your react components a little. having everything on one index.jsx file seems to clutter up a lot of your logic

File: client/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+          </div>
```
Comment (cshih1141) [inline client/src/index.jsx]: this seems to be the exact same code as line 126 - 130. think about pulling things like this out into its own component and then you can call that component in both locations if it needs to be in two places. That way you won't run into issues where you accidentally change one and not the other if things need to be edited.

File: client/src/index.jsx (hunk 2 above)
Comment (cshih1141) [reply client/src/index.jsx]: Theres a bunch of code thats repeated throughout, this was just an example.

File: client/src/index.jsx (hunk 3 above)
Comment (galw) [reply client/src/index.jsx]: Thanks for the input Craig, that's my plan after I the testing according to our next deliverable. I plan on brake everything into small react components and have another pull request for that.

File: client/src/index.jsx (hunk 2 above)
Comment (galw) [reply client/src/index.jsx]: Same as previous, that's my plan for later after I do the testing according to our next deliverable. I plan on brake everything into small react components.

File: public/index.html (hunk 1 above)
Comment (galw) [reply public/index.html]: After I finish building my react component I will either leave it black or with a little small message.

**Pull request merged**